Sources/Logging/Logging.swift

```diff
@@ -157,13 +169,13 @@ private class MUXLogHandler: LogHandler {
         }
     }
 
-    public func log(level: LogLevel, message: String, file: String, function: String, line: UInt) {
+    public func log(level: Logging.Level, message: String, file: String, function: String, line: UInt) {
         self.handlers.forEach { handler in
             handler.log(level: level, message: message, file: file, function: function, line: line)
         }
     }
 
-    public var metadata: LoggingMetadata? {
+    public var metadata: [String: Logging.MetadataValue]? {
```

Comment:
personally i like the typealias, but just a personal preference

Reply:
oh yes, I can bring it back

Reply:
done